server/sonar-server/src/main/java/org/sonar/server/es/NewIndex.java

```diff
@@ -235,6 +236,17 @@ public StringFieldBuilder enableGramSearch() {
     }
 
     /**
+     * Norms consume useless memory if string field is used for filtering or aggregations.
+     *
+     * https://www.elastic.co/guide/en/elasticsearch/reference/2.3/norms.html
+     * https://www.elastic.co/guide/en/elasticsearch/guide/current/scoring-theory.html#field-norm
+     */
+    public StringFieldBuilder disableNorms() {
```

Comment:
From what I see, norms are mot of the time disabled on string fields. Shouldn't 'disabled' be the default value ?